### PR TITLE
Remove warning for unknown functions - part 1

### PR DESCRIPTION
**Pull request by matsl**
## What

Remove warning for unknown functions - part 1

## Why

Warnings should be removed. The part 1 is because this is what I managed to do today.

### CODE REVIEW

File: hargs.el [hunk 1]
```diff
@@ -39,6 +39,11 @@
 (add-hook 'completion-setup-hook #'hargs:set-string-to-complete)
 (add-hook 'minibuffer-exit-hook  #'hargs:unset-string-to-complete)
 
+;;; ************************************************************************
+;;; Public declarations
+;;; ************************************************************************
+(declare-function monkey-filename "etx:monkey")
```
Comment (rswgnu) [inline hargs.el:45]: Typo: 'ext'.

File: hmouse-drv.el [hunk 2]
```diff
@@ -109,6 +109,17 @@ Default is nil."
   "Used to pass the value of a selected region between a Smart Key depress and release.
 This permits the Smart Keys to behave as paste keys.")
 
+;;; ************************************************************************
+;;; Public declarations
+;;; ************************************************************************
+(declare-function br-in-view-window-p "ext:oobr")
+(declare-function br-next-listing-window  "ext:oobr")
+(declare-function br-to-view-window  "ext:oobr")
```
Comment (rswgnu) [inline hmouse-drv.el:117]: These should all be "ext:br" as they are in br.el of the OO-Browser.

File: hmouse-tag.el [hunk 3]
```diff
@@ -128,6 +128,12 @@ Keyword matched is grouping 1.  Referent is grouping 2.")
   :type '(file :must-match t)
   :group 'hyperbole-commands)
 
+;;; ************************************************************************
+;;; Public declarations
+;;; ************************************************************************
+(declare-function br-edit "ext:oobr")
```
Comment (rswgnu) [inline hmouse-tag.el:134]: "ext:br"

File: hmouse-tag.el [hunk 4]
```diff
@@ -128,6 +128,12 @@ Keyword matched is grouping 1.  Referent is grouping 2.")
...
+(declare-function br-edit-feature "ext:oobr")
```
Comment (rswgnu) [inline hmouse-tag.el:135]: "ext:br-ftr"

File: hui-mouse.el [hunk 5]
```diff
@@ -45,6 +45,8 @@
   (require 'hmouse-tag))
 (require 'hsys-org)
 
+(require 'tar-mode)
```
Comment (rswgnu) [inline hui-mouse.el:48]: (eval-when-compile (require 'tar-mode))

File: hbdata.el [hunk 6]
```diff
@@ -50,6 +50,7 @@
 ;;; ************************************************************************
 
 (require 'hbmap)
+(require 'hgnus)
```
Comment (rswgnu) [inline hbdata.el:53]: See if (require 'hmail) will work here instead, as it does not have all the dependencies of hgnus.el.  You are going to want (require 'hrmail) too at some point but that also has a lot of dependencies so I wouldn't add that right now but just declare the functions from there that you need.

**Review by rswgnu — CHANGES_REQUESTED** [review]
Consider this approved after the changes requested inline are made, so you can push.  -- Bob

File: hbdata.el (hunk 6 above)
Comment (rswgnu) [reply hbdata.el:53]: Good question.  I would just leave them for now and lets get these added to the open Todo items and then I can look at it later.  Please merge if you feel ready.

Comment (matsl) [reply hbdata.el:53]: I will merge and move on to part2 in a new PR! :-)

**Pull request merged**